### PR TITLE
Disable failing linux tests

### PR DESCRIPTION
**Pull request by cdmihai**

### CODE REVIEW

**Comment by rainersigwald** [comment]
LGTM. Do we have an explicit issue for the some-tests-fail-and-are-generically-disabled-but-should-at-the-very-least-be-specifically-disabled-with-a-bug-number situation?

**Comment by cdmihai** [comment]
@dotnet-bot test  OSX Build please

**Comment by cdmihai** [comment]
@dotnet-bot test OSX Build please

File: dir.targets [hunk 1]
```diff
@@ -36,6 +36,8 @@
     <XunitOptions Condition="'$(OS)'=='Unix'">$(XunitOptions) -notrait category=nonlinuxtests</XunitOptions>
     <XunitOptions Condition="'$(OS)'=='OSX'">$(XunitOptions) -notrait category=nonosxtests</XunitOptions>
     <XunitOptions Condition="'$(OS)'=='OSX' and '$(NetCoreBuild)'=='true'">-notrait category=nonosxtests -notrait category=netcore-osx-failing</XunitOptions>
+    <XunitOptions Condition="'$(OS)'=='Unix' and '$(NetCoreBuild)'=='true'">-notrait category=nonlinuxtests -notrait category=netcore-linux-failing</XunitOptions>
```
Comment (radical) [inline dir.targets]: Use `$(XunitOptions)` - 
`<XunitOptions Condition="'$(OS)'=='Unix' and '$(NetCoreBuild)'=='true'">$(XunitOptions) -notrait category=netcore-linux-failing</XunitOptions>` 

And same for the `OSX && netcore` case.

**Comment by cdmihai** [comment]
@dotnet-bot test this please

**Comment by cdmihai** [comment]
@dotnet-bot test this please

**Comment by cdmihai** [comment]
@dotnet-bot test Ubuntu Build please

**Comment by cdmihai** [comment]
The failing tests do not appear in the xunit xml test reports :(
Looking into the issue.

In the meantime, linux CI jobs will fail. Should I revert the netci.groovy commit to master until I fix this?

**Comment by rainersigwald** [comment]
@cdmihai 

> Should I revert the netci.groovy commit to master until I fix this?

I say yes, but I don't feel too strongly about it.

**Comment by cdmihai** [comment]
This is rather stale. Should be redone from scratch. Closing.

**Pull request closed without merging**